docs/definitions.dox

```diff
@@ -258,6 +264,14 @@
  * \}
  *
  *
+ * \defgroup Syscon Syscon
+ *  Syscon processor communication
+ * \{
+ *     \defgroup SceSyscon Syscon communication Library
```

Comment:
Maybe put it in the `Lowio Low-level I/O` group instead? I would prefer to avoid to create a new root group just for Syscon.